code/modules/research/designs/spacepod_designs.dm

```diff
@@ -0,0 +1,318 @@
+	desc = "Allows for the construction of spacepod armor. This is the syndicate version."
+	id = "podarmor_dark"
+	build_type = PODFAB
+	req_tech = list("materials" = 6, "engineering" = 6, "combat" = 6, "syndicate" = 6)
+	build_path = /obj/item/pod_parts/armor/syndicate
+	category = list("Pod Armor")
+	materials = list(MAT_METAL=15000,MAT_GLASS=5000,MAT_PLASMA=10000,MAT_DIAMOND=5000,MAT_SILVER=7500)
+
+
+/datum/design/pod_armor_gold
+	construction_time = 400 //more time than frames, less than pod core
+	name = "Pod Armor (golden)"
+	desc = "Allows for the construction of spacepod armor. This is the golden version."
+	id = "podarmor_gold"
+	build_type = PODFAB
+	req_tech = list("materials" = 5, "engineering" = 3) //Slightly higher than gold, due to the requirement of getting actual protection from gold.
+	build_path = /obj/item/pod_parts/armor/gold
+	category = list("Pod Armor")
+	materials = list(MAT_METAL=5000,MAT_GLASS=2500,MAT_PLASMA=7500,MAT_GOLD=10000)
+
+//////////////////////////////////////////
+//////SPACEPOD GUNS///////////////////////
+//////////////////////////////////////////
+
+/datum/design/pod_gun_disabler
+	construction_time = 200
+	name = "Spacepod Equipment (Disabler)"
+	desc = "Allows for the construction of a spacepod mounted disabler."
+	id = "podgun_disabler"
+	build_type = PODFAB
+	req_tech = list("materials" = 2, "combat" = 2)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/disabler
+	category = list("Pod Weaponry")
+	materials = list(MAT_METAL = 15000)
+
+/datum/design/pod_gun_disabler
+	construction_time = 200
+	name = "Spacepod Equipment (Burst Disabler)"
+	desc = "Allows for the construction of a spacepod mounted disabler. This is the burst-fire model."
+	id = "podgun_bdisabler"
+	build_type = PODFAB
+	req_tech = list("materials" = 3, "combat" = 3)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/burst_disabler
+	category = list("Pod Weaponry")
+	materials = list(MAT_METAL = 15000,MAT_PLASMA=2000)
+
+/datum/design/pod_gun_laser
+	construction_time = 200
+	name = "Spacepod Equipment (Laser)"
+	desc = "Allows for the construction of a spacepod mounted laser."
+	id = "podgun_laser"
+	build_type = PODFAB
+	req_tech = list("materials" = 3, "combat" = 3, "plasmatech" = 2)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/laser
+	category = list("Pod Weaponry")
+	materials = list(MAT_METAL=10000,MAT_GLASS=5000,MAT_GOLD=1000,MAT_SILVER=2000)
+
+/datum/design/pod_mining_laser_basic
+	construction_time = 200
+	name = "Basic Mining Laser"
+	desc = "Allows for the construction of a weak mining laser"
+	id = "pod_mining_laser_basic"
+	req_tech = list("materials" = 3, "powerstorage" = 2, "engineering" = 2, "magnets" = 3, "combat" = 2)
+	build_type = PODFAB
+	materials = list(MAT_METAL = 10000, MAT_GLASS = 5000, MAT_SILVER = 2000, MAT_URANIUM = 2000)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/mining_laser_basic
+	category = list("Pod Weaponry")
+
+/datum/design/pod_mining_laser_basic
```

Comment:
Uncorrected copy-paste of the above design. Also, Kinetic Accelerators are not lasers. Please update all "mining_laser" paths to reflect that they are actually Kinetic Accelerators.